src/app/src/test/kotlin/org/imperial/mrc/hint/database/ProjectRepositoryTests.kt

```diff
@@ -180,10 +178,10 @@ class ProjectRepositoryTests
                 .where(PROJECT.ID.eq(projectId))
                 .fetchOne()
 
-        assertThat(project[PROJECT.USER_ID]).isEqualTo(uid)
-        assertThat(project[PROJECT.NAME]).isEqualTo("testProjectRepo")
-        assertThat(project[PROJECT.SHARED_BY]).isEqualTo(uid)
-        assertThat(project[PROJECT.NOTE]).isEqualTo("test note")
+        assertThat(project?.get(PROJECT.USER_ID)).isEqualTo(uid)
```

Comment:
No longer supported, old code was giving compiler error.

Reply:
weird that it's working in the repos then?

Reply:
Oh, is it because `fetchOne` now returns a nullable? I noticed you were throwing if none found in the repo code, and I guess that explains the change required here.